src/Roslyn/Conventional.Roslyn/Conventional.Roslyn.csproj

```diff
@@ -30,6 +30,21 @@
     <ItemGroup>
         <None Update="tools\*.ps1" CopyToOutputDirectory="Always" Pack="true" PackagePath="tools" />
         <None Include="$(OutputPath)\Conventional.Roslyn.Analyzers.dll" Pack="true" PackagePath="analyzers/dotnet/cs" Visible="false" />
+        
     </ItemGroup>
 
+    <PropertyGroup>
```

Comment:
I took a dive down the rabbit hole to find out why this required so much msbuild chicanery to work.

https://github.com/dotnet/sdk/issues/6688

It is a bit of a sad state that you need to go to these lengths to package a project dependency in a nuget package 😢

Reply:
Yeah, I've found that workaround some time ago on one of my projects, quite sad to see this is still the case.